### PR TITLE
Allows you to deconstruct canisters

### PR DESCRIPTION
**Pull request by ReoDaProtovali**
Makes it so that you can deconstruct empty or broken gas canisters. Simple as that.
dreamseeker_tNR0MiSr3D

### CODE REVIEW

**Comment by Screemonster** [comment]
holy shit finally

File: code/game/machinery/atmoalter/canister.dm [hunk 1]
```diff
@@ -239,6 +239,22 @@ update_flag
 	..()
 
 /obj/machinery/portable_atmospherics/canister/attackby(var/obj/item/weapon/W as obj, var/mob/user as mob)
+	if(istype(W, /obj/item/weapon/weldingtool)) //Vorestart: Deconstructable Canisters
+		var/obj/item/weapon/weldingtool/WT = W
+		if(!WT.remove_fuel(0, user))
+			to_chat(user, "The welding tool must be on to complete this task.")
+			return
+		if(air_contents.return_pressure() > 1 && !destroyed)
+			to_chat(user, "<span class ='warning'>\The [src]'s internal pressure is too high! Empty the canister before attempting to weld it apart.")
```
Comment (Very-Soft) [inline code/game/machinery/atmoalter/canister.dm:248]: should be

to_chat(user, "<span class ='warning'>\The [src]'s internal pressure is too high! Empty the canister before attempting to weld it apart.</span")

Comment (KillianKirilenko) [reply code/game/machinery/atmoalter/canister.dm:248]: Or it needs a closing `</span>` tag, either should fix CI.

I've made the same mistake a couple times.

Comment (ReoDaProtovali) [reply code/game/machinery/atmoalter/canister.dm:248]: OOPS

**Comment by ReoDaProtovali** [comment]
missing </span> should exist now

**Pull request merged**